### PR TITLE
add content for rule grub enable fips mode rhel6

### PR DESCRIPTION
**Pull request by cyarbrough76**
#### Description:

- _add content for rule grub_enable_fips_mode in rhel6_

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Rule rhel6:
 Bash remediation is newly added.
 The rule doesn't occur in any profile nor product.
Profile stig on rhel6:
 Rule grub_enable_fips_mode added to stig profile.

**Recommended tests to execute:**
 build_product rhel6
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel6-ds.xml stig

File: linux_os/guide/system/bootloader-grub-legacy/grub_enable_fips_mode/rule.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+rationale: |-
+    Use of weak or untested encryption algorithms undermines the purposes of using encryption 
+    to protect data. The operating system must implement cryptographic modules adhering to the 
+    higher standards approved by the federal government to ensure the algorithms have been tested 
+    and validated.
+
+severity: high
+
+identifiers:
+    cce@rhel6: 26911-8
```
Comment (matejak) [inline linux_os/guide/system/bootloader-grub-legacy/grub_enable_fips_mode/rule.yml:30]: Please take another (random) CCE from `shared/references/cce-redhat-avail.txt` to make the test pass.

**Comment by redhatrises** [comment]
As this rule already exists, I am not seeing the need for adding this rule. If anything, the existing rule should be added to the profile instead.

**Comment by matejak** [comment]
A combination of no activity, slight controversy and product in a very late maintenance phase suggests that the PR should be closed.

**Pull request closed without merging**